harness/determined/common/experimental/experiment.py

```diff
@@ -87,6 +87,8 @@ class Experiment:
             is 100% completion.
         description: (Mutable, string) Description of the experiment.
         notes: (Mutable, str) Notes for the experiment.
+        project_id: (Mutable, int) ID of the project the experiment is a part of.
```

Comment:
project_id: (Mutable, int) The ID of the project associated with the experiment.

Reply:
I feel there's some information in "project is a part of an experiment" and "experiment runs in a workspace." These requested changes remove that. Is my conception of project/workspace/experiment wrong?